### DISCUSSION TITLE
Stream do canal NOW

### DISCUSSION THREAD

**Discussion by psads-git** [LITUATUI/M3UPT#98] [General]
Bom dia,

Acredito que, provavelmente, não existe _stream_ do canal NOW, mas, no caso de existir, parece-me que, porventura, seria uma boa ideia acrescentá-lo à lista.

Obrigado!

**Comment by LITUATUI** [comment]
Boa tarde.

Não havendo um stream público e oficial não é possível adicionar à lista pública.
